projects/components/src/navigation/navigation-list.component.ts

```diff
@@ -29,7 +38,12 @@ import { FooterItemConfig, NavItemConfig, NavItemLinkConfig, NavItemType } from
             <hr *ngSwitchCase="'${NavItemType.Divider}'" class="nav-divider" />
 
             <ng-container *ngSwitchCase="'${NavItemType.Link}'">
-              <ht-nav-item [config]="item" [active]="item === activeItem" [collapsed]="this.collapsed"></ht-nav-item>
+              <ht-nav-item
+                (click)="this.setPageTimeRangeForSelectedNavItem(item)"
```

Comment:
Now that I think about it - navigation-list is a common, reused component. The behavior we're trying to add here is unique to the left nav isn't it?

Reply:
yeah, We should emit these click events to the parent list and there we can call the set time range method

That also means that `resolveFeaturesAndUpdateVisibilityForNavItems` should be called in the parent component